node-scripts/spec/pipeline3/impl/docker.spec.js

```diff
@@ -0,0 +1,296 @@
+"use strict";
```

Comment:
Probably should be renamed **docker-monolith.spec.js** right?

Reply:
this is renamed to **dockerMonolith.spec.js** as well as **dockerCompose.spec.js**